test/integration/nxos.yaml

```diff
@@ -298,35 +298,45 @@
           name: nxos_logging
         when: "limit_to in ['*', 'nxos_logging']"
       rescue:
-        - set_fact: test_failed=true
+        - set_fact:
+            failed_modules: "{{ failed_modules }} + [ 'nxos_ip_interface' ]"
+            test_failed: true
 
     - block:
       - include_role:
           name: nxos_bgp
         when: "limit_to in ['*', 'nxos_bgp']"
       rescue:
-        - set_fact: test_failed=true
+        - set_fact:
+            failed_modules: "{{ failed_modules }} + [ 'nxos_ip_interface' ]"
```

Comment:
Shouldn't this be `nxos_bgp` instead of `nxos_ip_interface`?  Same problem below

Reply:
Other than these, it looks good to me.